main.py

```diff
@@ -2,6 +2,38 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    lists_genre = ["Western", "Action", "Dramma"]
```

Comment:
The genre list contains a typo: 'Dramma' should be 'Drama'. However, this is corrected later in the code, so it might be intentional for demonstration purposes.